recipe/meta.yaml

```
@@ -1,5 +1,7 @@
 {% set name = "ptxcompiler" %}
 {% set version = "0.3.0" %}
+{% set cuda_version = '.'.join(environ.get('CUDA', '11.6').split('.')[:2]) %}
+{% set cuda_major = cuda_version.split('.')[0] %}
```

Think we can rewrite this with `cuda_compiler_version`

```suggestion
{% set cuda_major = environ.get("cuda_compiler_version", "11.6").split(".")[0] %}
```